tests/court_tests/test_court.py

```diff
@@ -0,0 +1,64 @@
+import pytest
+from court.models import Court
+
+
+TEST_ID = 1
+TEST_X = 1.234567
+TEST_Y = 9.876543
+TEST_CITY = 'New York'
+TEST_NEIGHBORHOOD = 'Hogwarts'
+TEST_MAX_PLAYERS = 5
+
+
+@pytest.fixture
+def court():
+    return Court(id=TEST_ID, x=TEST_X, y=TEST_Y, city=TEST_CITY, neighborhood=TEST_NEIGHBORHOOD,
+                 max_players=TEST_MAX_PLAYERS)
+
+
+@pytest.mark.django_db
+class TestCourtModel:
+
+    def test_court_model_save_method(self):
+        court = Court.objects.create(
+            id=TEST_ID,
+            x=TEST_X,
+            y=TEST_Y,
+            city=TEST_CITY,
+            neighborhood=TEST_NEIGHBORHOOD,
+            max_players=TEST_MAX_PLAYERS
+        )
+        assert court.id == TEST_ID
+        assert court.x == TEST_X
+        assert court.y == TEST_Y
+        assert court.city == TEST_CITY
+        assert court.neighborhood == TEST_NEIGHBORHOOD
+        assert court.max_players == TEST_MAX_PLAYERS
+
+    def test_court_model_primary_key_field(self):
+        court = Court(id=1)
+        assert court.id == 1
+
+    def test_court_model_x_field(self):
+        court = Court(x=1.234567)
+        assert court.x == 1.234567
+
+    def test_court_model_y_field(self):
+        court = Court(y=9.876543)
+        assert court.y == 9.876543
+
+    def test_court_model_city_field(self):
+        court = Court(city='New York')
+        assert court.city == 'New York'
+
+    def test_court_model_neighborhood_field(self):
+        court = Court(neighborhood='Hogwarts')
+        assert court.neighborhood == 'Hogwarts'
+
+    def test_court_model_max_players_field(self):
+        court = Court(max_players=5)
+        assert court.max_players == 5
+
+    def test_delete_court(self, court):
+        court.delete()
+        assert court not in Court.objects.all()
```

Comment:
You didn't check that it was not there in the first place, so how can you tell if it worked? I suspect it's not there as you didn't save it to the db.